SWADroid/src/main/res/values-es/lists.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+    <string-array name="gameMenuItems">
+        <item>Juegos activos</item>
+    </string-array>
```

Comment:
A menu with only one item does not make sense. Maybe you can simplify the design of the UI to eliminate it.